.github/workflows/phpunit.yml

```diff
@@ -70,7 +74,11 @@ jobs:
       run: php tests/phpinfo.php
 
     - name: Start broker service
-      run: docker start ${{ job.services.rabbitmq.id }}
+      run: docker restart ${{ job.services.rabbitmq.id }}
+
+    - name: Configure for PHP 8.1
+      run: composer config platform.php 8.0.99
+      if: matrix.php == '8.1'
```

Comment:
Let's wait for phpspec/phpspec compatible version. There is no rush for PHP8.1 as it wasn't released yet.

Reply:
OK, let's wait, but not too long. Please note that real projects contains hundreds of dependencies, and all of them should be fixed and tested, as well as the projects themself. It's a good practice to be ready for the new PHP version **before** its release, not after. Otherwise the library becomes a blocker for everyone who uses it.